behat/symfony2-extension/2.1/behat.yaml.dist

```diff
@@ -0,0 +1,7 @@
+default:
+    extensions:
+        Behat\Symfony2Extension:
+            kernel:
+                bootstrap: vendor/autoload.php
+                path: src/Kernel.php
```

Comment:
this is useless as well. The Kernel class is autoloadable since a long time (we changed it in symfony-standard 2.8) so there is no need to make the Behat extension require the kernel file directly. The autoloader will handle it.